weldx/schemas/weldx.bam.de/weldx/datamodels/multi_pass_weld-0.1.0.yaml

```diff
@@ -0,0 +1,94 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://weldx.bam.de/weldx/schemas/datamodels/multi_pass_weld-0.1.0"
+
+title: |
+  A generic multi layer GMAW weldment.
+
+type: object
+properties:
+    workpiece:
+      description: |
+        The workpiece to be welded defined by the base metal and the geometric description of the weld seam.
+      type: object
+
+    weld_seam:
+      description: |
+        List of weld seams composing the final weldment.
+      type: array
+      items:
+        $ref: "#/definitions/weld_seam"
+
+definitions:
+  weld_seam:
+    description: |
+      A single weld seam consisting of one or more layers.
+    type: array
+    items:
+      $ref: "#/definitions/weld_layer"
+
+  weld_layer:
+    description: |
+      A single weld layer consisting of one or more beads.
+    type: array
+    items:
+      $ref: "#/definitions/weld_bead"
+
+  weld_bead:
+    description: |
+      A single weld bead consisting of one or more weldments.
+    weld_layers:
+      type: array
+      items:
+        $ref: "#/definitions/weldment"
+
+  weldment:
+    description: |
+      Process metadata and measurements about a single weldment.
+    type: object
+    properties:
+      process:
+        $ref: "#/definitions/GMAW_process"
+      TCP:
+        description: |
+          Transformation describing the welding TCP movement in relation to the groove coordinates.
```

Comment:
I have taken this from the single_pass_weld schema but I think in general it would be more generic to describe the contact tube position and orientation (with an optional target stickout length)